### ISSUE TITLE
epyrun doesn't work

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10708]
|<img alt="marienz's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| marienz reported|
|-|-|
|Trac ID|trac#1067|
|Type|defect|
|Created|2005-05-30 17:02:37Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1067 1067
type__defect defect
reporter__marienz marienz
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1117472557000000 1117472557000000
changetime__1163376607000000 1163376607000000
version__ 
owner__ 
cc__spiv cc__marienz
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="marienz's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|marienz commented|
|-|-|

```
#!html
<pre>
running "admin/epyrun --debug" in a svn checkout from today with z.i 3.0.1 and
epyrun 2.1 installed gives me incomplete docs and the following traceback:

  Warning: Can't find module for 'IOldApplication'
                                     
  Warning: Can't find module for 'IOldApplication'
                                     
  Warning: Can't find module for 'IOldApplication'
Traceback (most recent call last):   
  File "admin/epyrun", line 437, in ?
    cli()                            
  File "/usr/lib/python2.4/site-packages/epydoc/cli.py", line 113, in cli
    if options['action'] == 'html': _html(docmap, options)
  File "/usr/lib/python2.4/site-packages/epydoc/cli.py", line 605, in _html
    try: html_doc.write(options['target'], progress.report)
  File "admin/epyrun", line 210, in write
    HTMLFormatter.write(self, directory, progress_callback)
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 475, in write
    progress_callback, uid.is_public(), uid, doc)
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 576, in _write
    write_func(public, private, *args)
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 801, in _write_class
    'Method Summary')                
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 1711, in
_write_func_summary
    function, container)             
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 1759, in
_write_func_summary_row
    fsum = self._summary(fdoc, fdoc.uid())
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 3020, in _summary
    summary = self._docstring_to_html(doc.summary(), uid, indent).strip()
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 2830, in
_docstring_to_html
    return docstring.to_html(linker, indent=indent)
  File "/usr/lib/python2.4/site-packages/epydoc/markup/epytext.py", line 1675,
in to_html
    self._html = self._to_html(self._tree, docstring_linker, indent)
  File "/usr/lib/python2.4/site-packages/epydoc/markup/epytext.py", line 1705,
in _to_html
    children = [self._to_html(c, linker, indent+2, seclevel)
  File "/usr/lib/python2.4/site-packages/epydoc/markup/epytext.py", line 1705,
in _to_html
    children = [self._to_html(c, linker, indent+2, seclevel)
  File "/usr/lib/python2.4/site-packages/epydoc/markup/epytext.py", line 1732,
in _to_html
    return linker.translate_identifier_xref(children[1], children[0])
  File "/usr/lib/python2.4/site-packages/epydoc/html.py", line 3462, in
translate_identifier_xref
    uid = findUID(identifier, self._container, self._docmap)
  File "/usr/lib/python2.4/site-packages/epydoc/uid.py", line 1090, in findUID
    uid = _find_object_in_module(name, container.value(), docmap)
  File "/usr/lib/python2.4/site-packages/epydoc/uid.py", line 1151, in
_find_object_in_module
    obj_uid = make_uid(obj)          
  File "admin/epyrun", line 173, in make_uid
    return make_uid_orig(obj, base_uid, shortname)
  File "/usr/lib/python2.4/site-packages/epydoc/uid.py", line 820, in make_uid
    raise TypeError('Cannot create a UID for a '+
TypeError: Cannot create a UID for a str without a base UID.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [18655]) Remove admin/epyrun

Author: exarkun
Reviewer: itamarst
Fixes [#10708](https://github.com/twisted/twisted/issues/10708)

admin/epyrun has long been broken and unused.  Remove it and the keybinding
for it in emacs' twisted-dev mode.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
